README.md

```diff
@@ -65,6 +65,21 @@ Code Overview
 
 These are the important parts of the Telemetry Server architecture.
 
+The simplified pipeline works as follows:
+
+1. Telemetry pings are generated by Firefox instances and are sent out from users' computers.
+    * To simulate this for testing purposes, run `python test/postGZ.py FOLDER_CONTAINING_PINGS` to submit all the pings in the specified folder.
+    * Sample pings can be found under one's Firefox profile, in the `saved-telemetry-pings` subfolder - they are human-readable JSON files.
+    * The `test/sample-saved_session-pings` folder can be used as an example: run `python test/postGZ.py test/test-saved_session-pings` while the Node server is running.
+2. The pings are received by a Node server (which can be found in `http/server.js`).
+    * Configure the server first by editing `http/server_config.json` - make sure to set the `log_path` (and ensure this folder exists) and `stats_log_file` options.
+    * To start the server locally, run `node server.js ./server_config.json`.
+    * The submitted pings are written to disk at the configured location.
+3. The server writes the pings to storage as fast as possible. Each ping written to disk is an LZMA compressed JSON file.
```

Comment:
This isn't quite true - pings are written to disk in batches (up to the configured max file size), and are written out in the same format they are submitted (either gzipped or raw, typically, but whatever bytes are sent are directly saved to these batch files).